src/components/PushBanner/PushBanner.spec.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import PushBanner from '.'
+import { makeDiskInfos } from 'cozy-client/dist/models/instance'
+import { usePushBannerContext } from './PushBannerProvider'
+import { isFlagshipApp } from 'cozy-device-helper'
+import useInstanceInfo from 'hooks/useInstanceInfo'
+
+jest.mock('./QuotaBanner', () => () => <div>QuotaBanner</div>)
+
+jest.mock('../pushClient/Banner', () => () => <div>BannerClient</div>)
+
+jest.mock('cozy-client/dist/models/instance', () => ({
+  makeDiskInfos: jest.fn()
+}))
+
+jest.mock('hooks/useInstanceInfo', () =>
+  jest.fn(() => ({
+    isLoaded: true
+  }))
+)
+
+jest.mock('./PushBannerProvider', () => ({
+  usePushBannerContext: jest.fn(() => ({
+    bannerDismissed: {}
+  }))
+}))
+
+jest.mock('cozy-device-helper', () => ({
+  isFlagshipApp: jest.fn(() => false)
+}))
+
+describe('PushBanner', () => {
+  const setup = (percentUsage = 50, dismissed = false) => {
+    usePushBannerContext.mockReturnValue({
+      bannerDismissed: {
+        quota: dismissed
+      }
+    })
+    makeDiskInfos.mockReturnValue({
+      percentUsage
+    })
+    return render(<PushBanner />)
+  }
+
+  describe('QuotaBanner', () => {
+    it('should show quota banner when disk usage has reach 80%', () => {
+      setup(80)
```

Comment:
extra-nit-dont-change-this-pls: 

`const setup = ({ percentUsage = 50, dismissed = false }) => {` leads to `setup({ percentUsage: 80, dismissed: true })` which is more readable than `setup(80, false)`

Reply:
Noted for next time